walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/Event.java

```diff
@@ -93,4 +95,18 @@ private static boolean equals(Event a, Event b) {
     // interface documentation, equals will check for set-equality across all set implementations.
     return a.title.equals(b.title) && a.when.equals(b.when) && a.attendees.equals(b.attendees);
   }
+
+  /**
+   * Returns true iff at leas one attendee in requestedAttendees participates in this Event 
```

Comment:
Here and everywhere else: End comments which are sentences with a fullstop (also for consistency).